src/components/Footer.tsx

```diff
@@ -1,15 +1,176 @@
-import React from 'react';
-import { Container } from 'styled-bootstrap-grid';
-import styled from 'styled-components';
+import React from "react";
```

Comment:
@nabeel- I feel like ESlint should have caught double quotes here, no?

Reply:
@mmahalwy @nabeel-  I think its better if we have prettier on the development level, as in during the coding time and it will auto format on save. 

It doesn't hurt to have prettier config, right?

Reply:
Uh oh, looks like the commit (https://github.com/quran/quran.com-frontend-next/commit/1c00ebd13a5fa0ae598f50ff6ed1dd1d7a98d13d) from the PR (https://github.com/quran/quran.com-frontend-next/pull/25) doesn't exist on master anymore. Was master force-pushed?

Reply:
> I think its better if we have prettier on the development level, as in during the coding time and it will auto format on save

@MistaPidaus Agreed! We had this in the commit/PR I mentioned above but it seems to have disappeared from master. There is also a [pre-commit hook](https://github.com/quran/quran.com-frontend-next/blob/02c634a32404912b4a53871a365bf663f7650547/package.json#L79-L86) that will auto-fix errors, too

Reply:
@nabeel- I see. It looks like it.

Please, no more force push. It will ruin the project.